package.json

```diff
@@ -1177,6 +1183,26 @@
         {
           "command": "abracadabra.splitIfStatement",
           "when": "editorLangId == vue"
+        },
+        {
+          "command": "abracadabra.splitMultipleDeclarations",
+          "when": "editorLangId == javascript"
+        },
+        {
+          "command": "abracadabra.splitMultipleDeclarations",
+          "when": "editorLangId == javascriptreact"
+        },
+        {
+          "command": "abracadabra.splitMultipleDeclarations",
+          "when": "editorLangId == typescript"
+        },
+        {
+          "command": "abracadabra.splitMultipleDeclarations",
+          "when": "editorLangId == typescriptreact"
+        },
+        {
+          "command": "abracadabra.splitMultipleDeclarations",
+          "when": "editorLangId == vue"
```

Comment:
👍 Looks good. I agree that this part is a bit tricky and requires a lot of boilerplate. We'll eventually automate this so it's less error-prone.